packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -270,12 +270,25 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       props.inputProps.onClick(ev as React.MouseEvent<HTMLInputElement>);
     }
   };
-  const _onInputChange = (value: string, composing?: boolean) => {
+  const _onInputChange = (value: string, composing?: boolean, resultItemsList?: T[]) => {
```

Comment:
Would this ever have a value?

Reply:
yes, it would, this is what is getting updated